kimchi/Cargo.toml

```diff
@@ -13,6 +13,10 @@ license = "Apache-2.0"
 path = "src/lib.rs"
 bench = false       # needed for criterion (https://bheisler.github.io/criterion.rs/book/faq.html#cargo-bench-gives-unrecognized-option-errors-for-valid-command-line-options)
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+jemalloc-ctl = { workspace = true, optional = true }
```

Comment:
I am a bit worried that the package seems to not be maintained anymore: https://github.com/gnzlbg/jemallocator.

Reply:
It seems there is this maintained fork: https://github.com/tikv/jemallocator.

Reply:
It seems to be the one recommended in the Rust book: https://nnethercote.github.io/perf-book/build-configuration.html#alternative-allocators